crates/miden-lib/asm/account_components/ecdsa_k256_keccak.masm

```diff
@@ -0,0 +1,42 @@
+# The MASM code of the Ecdsa K256 Keccak authentication Account Component.
+#
+# See the `AuthEcdsaK256Keccak` Rust type's documentation for more details.
+
+use miden::auth::ecdsa_k256_keccak
+use miden::active_account
+
+type BeWord = struct @bigendian { a: felt, b: felt, c: felt, d: felt }
+
+# CONSTANTS
+# =================================================================================================
+
+# The slot in this component's storage layout where the public key is stored.
+const PUBLIC_KEY_SLOT = 0
+
+#! Authenticate a transaction using the Ecdsa signature scheme.
```

Comment:
Similar nit as in one of the other comments: "Ecdsa" -> "ECDSA" (but only in comments).